utils/ratlib.py

```diff
@@ -128,3 +165,75 @@ def try_parse_uuid(suspect: str) -> UUID:
 
     else:
         return result
+
+
+# color functions
+def color(text: str, text_color: Colors, bg_color=Optional[Colors]) -> str:
+    """
+    Colorizes the given string with the specified color.
+    Args:
```

Comment:
Newline between function description and the next block, otherwise it will not be parsed correctly